### PR TITLE
Feature/#153729004 - Style header

### PR DESCRIPTION
**Pull request by michaelbretagne**
Instead of putting the header navigation outside the main body and be able to draw the bottom border on the entire width, I gave a relative position to "body" and absolute position to "header:after".
Less refactoring like that. Let me know if this approach works for you.

### CODE REVIEW

File: client/src/components/Header/Header.scss [hunk 1]
```diff
@@ -1,26 +1,37 @@
 @import '../../scss/partials/partials';
 
 .header {
-  padding: 0 60px;
-  margin-top: 60px;
+  padding: 20px 60px;
```
Comment (timdavish) [inline client/src/components/Header/Header.scss]: How about a lower padding top/bottom, like `10px`? This way it matches the height from the design screenshot

File: client/src/components/Header/Header.scss [hunk 2]
```diff
@@ -1,26 +1,37 @@
...
   display: grid;
   grid-template-columns: repeat(2, 1fr);
+
+  &:after {
+    content: '';
+    position: absolute;
+    width: 100%;
+    top: 84px;
```
Comment (timdavish) [inline client/src/components/Header/Header.scss]: To compensate for the above change, this decreases to `64px`.

File: client/src/components/Header/Header.scss (hunk 1 above)
Comment (michaelbretagne) [reply client/src/components/Header/Header.scss]: Ok, I will do that. I just felt that 15px look better :-) But I am not a designer so I will stick to the design screenshot ;-)

**Comment by sexta13** [comment]
Another possible solution,  I think might be better then defining the position relative in body is to define a new div (maybe LineComponent) with a fixed position.

something like:

   ```
 position: fixed;
    top: 64px;
    border-bottom: 2px solid #F9F9F9;
    left: 0;
    right: 0;
```

**Comment by bbmoz** [comment]
I think either of the approaches that you or @sexta13 proposed work fine! I just said to put it outside the main content because that seemed straightforward at the time.

**Comment by timdavish** [comment]
Whatever you want to do is fine @michaelbretagne 
I'm going to approve since either way is fine. Feel free to make changes and it can get re-approved

**Pull request merged**